public/js/test/utils/renderComponentFromFixture.js

```diff
@@ -7,7 +7,7 @@ const { fromJS, Map } = require("immutable");
 const { combineReducers } = require("redux");
 
 const fixtures = require("../fixtures");
-const configureStore = require("../../create-store");
+const configureStore = require("../../util/create-store");
```

Comment:
moved create-store into util to reduce the top level files. also seemed closer to the middleware